firmware/examples/Gpio/source/main.cpp

```diff
@@ -0,0 +1,33 @@
+#include "L1_Drivers/gpio.hpp"
+
```

Comment:
You are missing the debug_print.hpp include. Be sure to include it.

Reply:
Is there a reason why the other examples don't include the debug_print.hpp file when DEBUG_PRINT() is used?

Reply:
Oh, if that is the case, then it needs to include that. It may been included through another include, but it should be included.